aws/cf.go

```diff
@@ -148,6 +150,7 @@ func createStack(svc cloudformationiface.CloudFormationAPI, spec *stackSpec) (st
 			cfParam(parameterTargetGroupVPCIDParameter, spec.vpcID),
 			cfParam(parameterTargetTargetPortParameter, fmt.Sprintf("%d", spec.targetPort)),
 			cfParam(parameterListenerSslPolicyParameter, spec.sslPolicy),
+			cfParam(parameterIpAddressTypeParameter, spec.ipAddressType),
```

Comment:
You also need to define this parameter in the `aws/cf_template.go` and reference it on the LB resource.

Reply:
fixed